docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -266,7 +266,7 @@ Unit tests
 
 You can add unit tests for your module in ``./test/units/modules``. You must first setup your testing environment. In this example, we're using Python 3.5.
 
-- Install the requirements (outside of your virtual environment): ``$ pip3 install -r ./test/lib/ansible_test/_data/requirements/units.txt``
+- Install the requirements (outside of your virtual environment): ``$ pip3 install -r ./test/units/requirements.txt``
```

Comment:
The docs should be updated to encourage the use of the `--requirements`, `--venv` or `--docker` option instead of manual installation of the requirements files. There are multiple requirements files needed, so simply changing the path here isn't correct.